### PR TITLE
Change logos for journal with new branding

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: assets/img/patterns/organisms/elife-logo-full.svg [hunk 1]
```diff
@@ -1,39 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<svg viewBox="0 0 136 70" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 43.2 (39069) - http://www.bohemiancoding.com/sketch -->
-<!--  David woz 'ere  -->
```
Comment (thewilkybarkid) [inline assets/img/patterns/organisms/elife-logo-full.svg]: 😆

Comment (discodavey) [reply assets/img/patterns/organisms/elife-logo-full.svg]: Wait for the review request

**Comment by thewilkybarkid** [comment]
Wonder if it's worth going for more readable SVGs rather than relying on what Sketch produces (so like https://raw.githubusercontent.com/libero/pattern-library/master/source/images/libero-logo.svg).

**Comment by discodavey** [comment]
This is not to merged until the 16th March as this is the date the new logo goes live.

**Comment by discodavey** [comment]
I am assuming this is something for @NickDuf to do rather than us or are you talking about us re-factoring the file supplied to us from Nick?

**Comment by davidcmoulton** [comment]
SVG workflow is something that we haven't formally addressed. There is definitely an opportunity to make source improvements downstream of the file generation. Would be useful to have a conversation with the people generating the SVGs to agree future workflow, so that we have the freedom of action re code changes without compromising the update process.

File: assets/img/patterns/organisms/elife-logo-full.svg (hunk 1 above)
Comment (davidcmoulton) [reply assets/img/patterns/organisms/elife-logo-full.svg]:
```
    ..................................................
    :                    ......                      :
    :                 .:||||||||:.                   :
    :                /            \                  :
    :               (   o      o   )                 :
    :-------@@@@----------:  :----------@@@@---------:
    :                     `--'                       :
    :                                                :
```

File: source/_patterns/02-organisms/global/site-header.mustache [hunk 2]
```diff
@@ -4,11 +4,11 @@
       <a href="#maincontent" class="site-header__skip_to_content__link button button--default">Skip to Content</a>
     </div>
     <a href="{{homePagePath}}" class="site-header__logo_link">
-      <picture class="site-header__logo_link_image">
```
Comment (davidcmoulton) [inline source/_patterns/02-organisms/global/site-header.mustache]: Why has this been removed? The [sass that relates to it](https://github.com/elifesciences/pattern-library/blob/d86df0b92b6a8c517da42101500ddd798e85c97a/assets/sass/patterns/organisms/site-header.scss#L63) is still there. Should either remove the sass as well (if there's a good reason to), or don't remove either.

File: source/_patterns/02-organisms/global/site-header.mustache [hunk 3]
```diff
@@ -4,11 +4,11 @@
...
+      <picture>
         {{! 45.625em is 730px, which is $bkpt-site--medium }}
-        <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/elife-logo-full.svg{{/assetRewrite}}" type="image/svg+xml" media="(min-width: 45.625em)">
-        <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/elife-logo-symbol.svg{{/assetRewrite}}" type="image/svg+xml">
-        <img src="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/elife-logo-full-1x.png{{/assetRewrite}}" alt="eLife logo" class="site-header__logo_link"/>
+        <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/elife-logo-xs.svg{{/assetRewrite}}" type="image/svg+xml" media="(min-width: 45.625em)">
+        <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/organisms/journal-symbol.svg{{/assetRewrite}}" type="image/svg+xml">
```
Comment (davidcmoulton) [inline source/_patterns/02-organisms/global/site-header.mustache]: Where did the name change to `journal-symbol` come from? Curious why it's not a like-for-like replacement.

Comment (discodavey) [reply source/_patterns/02-organisms/global/site-header.mustache]: It was what was supplied to me. I will change the name

File: source/_patterns/02-organisms/global/site-header.mustache (hunk 2 above)
Comment (discodavey) [reply source/_patterns/02-organisms/global/site-header.mustache]: Sorry.. Should still be there

File: assets/sass/patterns/organisms/site-header.scss [hunk 4]
```diff
@@ -46,10 +46,10 @@
 }
 
 .site-header__logo_link {
-  background: url("../img/patterns/organisms/elife-logo-symbol-1x.png") 0 0 no-repeat;
+  background: url("../img/patterns/organisms/journal-symbol@1x.png") 0 0 no-repeat;
```
Comment (davidcmoulton) [inline assets/sass/patterns/organisms/site-header.scss]: Filename?

Comment (discodavey) [reply assets/sass/patterns/organisms/site-header.scss]: oops

**Pull request merged**